### DISCUSSION TITLE
Unable to upload image where the button for upload is a hidden element

### DISCUSSION THREAD

**Discussion by Neha-test-dev** [getgauge/taiko#1811] [Development]
Earlier I was able to upload image (though the element is visible, the element is given as style="display: none;" when I try to locate it ) using the below line of code:
await attach('C:/imageName', fileField({type:'file'},{ selectHiddenElements: true }));

This was working fine but I updated Taiko to v1.1.4 and from then on it's erroring out. Could anyone please provide any solution for this issue?

**Comment by NivedhaSenthil** [comment]
Please use older version while we fix this issue, https://github.com/getgauge/taiko/issues/1771

**Reply by Neha-test-dev** [reply]
Ok. Thank you for the update. Would you please mind sharing the version which this was working as expected? I unfortunately lost track of which version this was working.

**Reply by NivedhaSenthil** [reply]
1.1.0 should work fine

**Reply by Neha-test-dev** [reply]
Thank you. I have another issue with this above code (mentioning here again - await attach('C:/imageName', fileField({type:'file'},{ selectHiddenElements: true }));  ). I have two images to upload on the same page so when I'm trying to upload the second image, the code uploads the second image in place of the first image (overriding the first image). I tried indexing but looks like fileField doesn't accept that also it being hidden element.

**Reply by Neha-test-dev** [reply]
@NivedhaSenthil Could you please let me know if we have anyway of indexing to upload image into the second index for the above mentioned code? Looks like elements and indexing works for lists and if we use xpaths. But I'm looking for indexing with attach action. 
Thank you in advance!
